pcsx2/CDVD/CDVD.cpp

```diff
@@ -920,25 +920,35 @@ void cdvdReset()
 	cdvd.ReadTime = cdvdBlockReadTime(MODE_DVDROM);
 	cdvd.RotSpeed = cdvdRotationTime(MODE_DVDROM);
 
+	ReadOSDConfigParames();
+
+	// Print timezone offset, DST, date format, and time format.
+	DevCon.WriteLn(Color_StrongGreen, configParams1.timezoneOffset < 0 ? "Timezone Offset: GMT%03d:%02u" : "Timezone Offset: GMT+%02d:%02u",
+				   configParams1.timezoneOffset / 60, configParams1.timezoneOffset % 60);
```

Comment:
Have you tested this with a number like -390 minutes?  I don't think this will do what you want.

Also you can use `%+03d` to force a sign to be printed, which might be a bit easier than messing with the size depending on whether it's positive or negative.